content/faq/libraries-frameworks/devops.mdx

```diff
@@ -51,8 +51,23 @@ showToc: true
 
 ## CI/CD
 
-- Azure DevOps Pipelines
-- Git Actions
+- On-Cloud
```

Comment:
Instead of using nested list, could you please use `#` as a header of each group and use only 1 level depth list item.

After you've fixed this, I will merge your PR. 
Thanks